olive/passes/pass_config.py

```diff
@@ -4,7 +4,9 @@
 # --------------------------------------------------------------------------
 from enum import Enum
 from pathlib import Path
-from typing import Callable, Dict, Optional, Type, Union
+from typing import Callable, Dict, Optional, Set, Type, Union
+
+from pydantic import Field
```

Comment:
this needs to be imported from `olive.common.pydantic_v1` like in line 12 so that we get the correct `Field` from v1 api.